mars/promise.py

```diff
@@ -28,6 +28,7 @@
 
 logger = logging.getLogger(__name__)
 _promise_pool = dict()
+_pack_qword = struct.Struct('<Q').pack
```

Comment:
How about the name `_pack_longlong`? What does `qword` here mean?

Reply:
``qword`` means quad bytes. This name consists with ``<Q``.